### PR TITLE
op-guide: fix tls related document

### PR DESCRIPTION
**Pull request by Connor1996**
Fix wrong config of tls document

### CODE REVIEW

**Comment by Connor1996** [comment]
PTAL @breeswish

File: sql/encrypted-connections.md [hunk 1]
```diff
@@ -5,6 +5,56 @@ category: user guide
 
 # 使用加密连接
 
+## 摘要
```
Comment (breeswish) [inline sql/encrypted-connections.md]: Looks like this content is already covered by later content?

Comment (Connor1996) [reply sql/encrypted-connections.md]: For what I mean, this part is much like TL;DR

Comment (breeswish) [reply sql/encrypted-connections.md]: But looks it's still too long 🤔🤣 I have never seen a TL;DR in so many lines

**Comment by Connor1996** [comment]
PTAL @breeswish

**Comment by breeswish** [comment]
Shall we also fix it for eng docs? @CaitinChen

**Comment by Connor1996** [comment]
>若要使 MySQL 客户端验证 TiDB 服务端身份，TiDB 服务端需至少配置 ssl-cert 和 ssl-key 参数，客户端需至少指定 --ssl-ca 参数，且 --ssl-mode 至少为 **VERIFY_IDENTITY**

seems that this part should changed also? @breeswish 
> Host name identity verification with VERIFY_IDENTITY does not work with self-signed certificates created automatically by the server, or manually using mysql_ssl_rsa_setup (see Section 6.4.3.1, “Creating SSL and RSA Certificates and Keys using MySQL”). Such self-signed certificates do not contain the server name as the Common Name value.

once we use VERIFY_IDENTITY for the certs generated by `mysql_ssl_rsa_setup`, it won't work either.

**Comment by breeswish** [comment]
@Connor1996 Agree! Looks like it should be "at least VERIFY_CA"? VERIFY_IDENTITY is the highest level.

**Comment by Connor1996** [comment]
@breeswish The default mode is `PREFERRED`, so I think the `at least xxx` is not necessary. 
see https://dev.mysql.com/doc/refman/5.7/en/encrypted-connection-options.html#option_general_ssl-mode

**Comment by breeswish** [comment]
@Connor1996 Seems that preferred is the lowest level ssl-mode (i.e. there can be no ssl at all), while verify_ca is the highest (most strict). "At least" is to express that there are multiple TLS modes available and some provide better security but *at least* this mode should be used to gain the desired security feature (in the context, it is client verification).

**Pull request merged**